test/valgrind/suppressions/ruby.supp

```diff
@@ -14,3 +14,12 @@
   fun:memmove
   fun:flo_to_s
 }
+{
+  <make_valid_false_positive>
```

Comment:
I don't think this is false positive (it never is tbh ^^)

Reply:
I merged in your PR with the upgraded memcheck functionality so this should be able to be removed now once `main` is merged in with this branch.